api/src/main/java/com/community/api/endpoint/avisoft/controller/Gender/GenderController.java

```diff
@@ -28,7 +28,7 @@ public ResponseEntity<?> getAllGender() {
         try {
             List<CustomGender> customGenderList = genderService.getAllGender();
             if (customGenderList.isEmpty()) {
-                return ResponseService.generateErrorResponse("NO GENDER IS FOUND", HttpStatus.NOT_FOUND);
+                return ResponseService.generateSuccessResponse("NO GENDER FOUND", new Object(), HttpStatus.OK);
```

Comment:
return ResponseService.generateErrorResponse("NO GENDER IS FOUND", HttpStatus.OK); will be more efficient to use